code/modules/organs/internal/organ_internal_vr.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/organ/internal/eyes/emp_act(severity)
+	if(!(robotic >= ORGAN_ASSISTED))
+		return
+	owner.eye_blurry += (4/severity)
+
+/obj/item/organ/internal/cell/emp_act(severity)
+	if(!(robotic >= ORGAN_ASSISTED))
```

Comment:
This will never be NOT robotic. It's not like you get this if you pick mechanical heart or anything like eyes might be.